---
permalink: /
title: "About me"
excerpt: "Hongbo Liu"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
<base target="_blank">

I am a postdoctoral fellow in [Susztak Lab](https://www.med.upenn.edu/susztaklab) in the Departments of Genetics and Medicine at the University of Pennsylvania.
Since 2007, my research has mainly focused on the genomics and epigenomics of complex human diseases by developing scalable machine-learning methods and integrating 
multimodal genomic and epigenomic data. A central objective of my study is to understand the genetic basis and epigenetic regulation of complex human diseases, 
by integrating genome-wide association studies with transcriptomic and epigenomic analyses. 
These studies have answered population genetic questions about epigenome-mediated heritability and causal gene prioritization via epigenome.

Research interests
======
* Population Genetics
* Epigenetic Regulation
* Single cell Omics
* Complex Human Diseases


News
======
<html>
<body>

<div id="site_content_isoform">
<!--<p><h1 id="gene_title">Please Cite:</h1></p>-->
<!--<div><p><strong>Human whole blood mQTL study (<a href="https://www.pnas.org/content/117/46/29013"><i>Sheng, X. et al.</i>, <i>Proceedings of the National Academy of Sciences</i>, 2020</a>):</strong></p>-->
<!--<p><i>Sheng, X. et al.</i> Systematic integrated analysis of genetic and epigenetic variation in diabetic kidney disease. <i>Proceedings of the National Academy of Sciences</i> (2020).</p>-->
<h1><font size="+2" color="black" id="title1"><strong>Publication related to the human kidney meQTL atlas:</strong></font></h1>
<p>Please cite following paper:</p><hr />
<p></p>
<table id="susztakmenu">
<tr>
    <td valign="bottom">
        <p><a href="https://www.nature.com/articles/s41588-022-01097-w" target="_blank"><strong>Epigenomic and transcriptomic analyses define core cell types, genes and targetable mechanisms for kidney disease.</strong></a></p>
        <p><i>Hongbo Liu, Tomohito Doke, Dong Guo, Xin Sheng, Ziyuan Ma, Joseph Park, Ha My T. Vy, Girish N. Nadkarni, Amin Abedini, Zhen Miao, Matthew Palmer, Benjamin F. Voight, Hongzhe Li, Christopher D. Brown, Marylyn D. Ritchie, Yan Shu and Katalin Susztak</i></p>
        <p><i>Nature Genetics 54, 950â€“962 (2022)</i></p>
    </td><!-- comment -->

<!--    <td valign="bottom"><div<a href="https://plu.mx/plum/a/?doi=10.1038%2Fs41588-022-01097-w" data-popup="bottom" data-size="large" class="plumx-plum-print-popup" data-site="plum" data-hide-when-empty="true"></a></div>
    </td>-->
    
    <td valign="bottom"><div class="altmetric-embed" data-badge-popover="bottom" data-badge-type="donut" data-altmetric-id="129787922"></div>
    </td>
 </tr>
 </table>

<p></p>
<p></p>
<p></p><hr />
<p></p>
<p></p>
<p>Link to the paper: <a href="https://www.nature.com/articles/s41588-022-01097-w" target="_blank"><i>Liu, H. et al.</i>, <i>Nature Genetics</i>, 2022</a></p>
<p></p>
<p>Supplementary Data: <a href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41588-022-01097-w/MediaObjects/41588_2022_1097_MOESM4_ESM.xlsx" target="_blank"><i>Nature Genetics</i></a> | <a href="https://doi.org/10.6084/m9.figshare.15183495" target="_blank"><i>figshare</i></a></p>
<p></p>
<p>Sequencing data on GEO database: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE200547" target="_blank"><i>GSE200547</i></a> | <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172008" target="_blank"><i>GSE172008</i></a></p>
<p></p>
<p>Customized code used in this study: <a href="https://github.com/hbliu/Kidney_Epi_Pri" target="_blank"><i>Github</i></a></p>

<p></p>
<p></p>
<p></p><hr />
<video width="1160" autoplay muted>
  <source src="image/NG_Paper.mp4" type="video/mp4">
</video>

</div>


<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Team work as always. Many thanks to my great mentor <a href="https://twitter.com/KSusztak?ref_src=twsrc%5Etfw">@KSusztak</a> and talented collaborators!<br>Hope everyone enjoy reading it! <a href="https://t.co/jYxREFeDYe">https://t.co/jYxREFeDYe</a></p>&mdash; Hongbo Liu (@Hongbo919Liu) <a href="https://twitter.com/Hongbo919Liu/status/1537473379418066944?ref_src=twsrc%5Etfw">June 16, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
</body>
</html>


<!-- 
Site-wide configuration
------
The main configuration file for the site is in the base directory in [_config.yml](https://github.com/academicpages/academicpages.github.io/blob/master/_config.yml), which defines the content in the sidebars and other site-wide features. You will need to replace the default variables with ones about yourself and your site's github repository. The configuration file for the top menu is in [_data/navigation.yml](https://github.com/academicpages/academicpages.github.io/blob/master/_data/navigation.yml). For example, if you don't have a portfolio or blog posts, you can remove those items from that navigation.yml file to remove them from the header. 

Create content & metadata
------
For site content, there is one markdown file for each type of content, which are stored in directories like _publications, _talks, _posts, _teaching, or _pages. For example, each talk is a markdown file in the [_talks directory](https://github.com/academicpages/academicpages.github.io/tree/master/_talks). At the top of each markdown file is structured data in YAML about the talk, which the theme will parse to do lots of cool stuff. The same structured data about a talk is used to generate the list of talks on the [Talks page](https://academicpages.github.io/talks), each [individual page](https://academicpages.github.io/talks/2012-03-01-talk-1) for specific talks, the talks section for the [CV page](https://academicpages.github.io/cv), and the [map of places you've given a talk](https://academicpages.github.io/talkmap.html) (if you run this [python file](https://github.com/academicpages/academicpages.github.io/blob/master/talkmap.py) or [Jupyter notebook](https://github.com/academicpages/academicpages.github.io/blob/master/talkmap.ipynb), which creates the HTML for the map based on the contents of the _talks directory).

**Markdown generator**

I have also created [a set of Jupyter notebooks](https://github.com/academicpages/academicpages.github.io/tree/master/markdown_generator
) that converts a CSV containing structured data about talks or presentations into individual markdown files that will be properly formatted for the academicpages template. The sample CSVs in that directory are the ones I used to create my own personal website at stuartgeiger.com. My usual workflow is that I keep a spreadsheet of my publications and talks, then run the code in these notebooks to generate the markdown files, then commit and push them to the GitHub repository.

How to edit your site's GitHub repository
------
Many people use a git client to create files on their local computer and then push them to GitHub's servers. If you are not familiar with git, you can directly edit these configuration and markdown files directly in the github.com interface. Navigate to a file (like [this one](https://github.com/academicpages/academicpages.github.io/blob/master/_talks/2012-03-01-talk-1.md) and click the pencil icon in the top right of the content preview (to the right of the "Raw | Blame | History" buttons). You can delete a file by clicking the trashcan icon to the right of the pencil icon. You can also create new files or upload files by navigating to a directory and clicking the "Create new file" or "Upload files" buttons. 

Example: editing a markdown file for a talk
![Editing a markdown file for a talk](/images/editing-talk.png)

For more info
------
More info about configuring academicpages can be found in [the guide](https://academicpages.github.io/markdown/). The [guides for the Minimal Mistakes theme](https://mmistakes.github.io/minimal-mistakes/docs/configuration/) (which this theme was forked from) might also be helpful.
 -->
