---
permalink: /
title: "About me"
excerpt: "Hongbo Liu"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---
<base target="_self">

Now, I am a Research Associate in [Susztak Lab](https://www.med.upenn.edu/susztaklab) in the Departments of Genetics and Medicine at the University of Pennsylvania.

Since 2007, my research has mainly focused on the genomics and epigenomics of age-related chronic diseases by developing scalable machine-learning methods and integrating 
multimodal genomic and epigenomic data. A central objective of my study is to identify disease-causal genes and understand their genetic and epigenetic regulation mechanisms, by integrating population genetic studies with transcriptomic and epigenomic analyses.
These studies have answered population genetic questions about epigenome-mediated heritability and causal gene prioritization via epigenome.


Research interests
======
* Population Genetics
* Epigenetic Regulation
* Single Cell Omics
* Age-Related Chronic Diseases
* Developmental Origins of Diseases


News
======

* $\colorbox{green}{{01/01/2024 Starting in March 2024, I'll be leading my own lab at the University of Rochester as an Assistant Professor. The research in my lab will focus on decoding the functional genome's role in aging and age-related diseases.}}$

* <span style="color:red">01/01/2024: We are actively seeking talented and passionate postdoctoral researchers (Computational Biology and Molecular Biology). If interested please send an email to Hongbo Liu (hongbo919@gmail.com) including your CV and a brief Research Statement with your background and interests.</span>


* 08/16/2022: In the latest study, I performed a comprehensive mapping of genetic architecture of kidney disease by integrating genomic analysis of GWAS (N=1.5 million), kidney eQTL (N=686), meQTL (N=443), snATACseq (N=60K cells), coloc, moloc, SMR, pheWAS. I proposed a multi-stage prioritization strategy and prioritized >500 kidney disease genes.
<html>
<body>
<video src="https://user-images.githubusercontent.com/2704756/210302639-8c4e8317-89e5-4a24-8160-dfe5bfb5d59d.mp4" loop="true" autoplay="autoplay" controls="controls" style="max-width: 730px;" muted>
</video>
</body>
</html>

Hongbo Liu, Tomohito Doke, Dong Guo, Xin Sheng, Ziyuan Ma, Joseph Park, Ha My T Vy, Girish N Nadkarni, Amin Abedini, Zhen Miao, Matthew Palmer, Benjamin F Voight, Hongzhe Li, Christopher D Brown, Marylyn D Ritchie, Yan Shu, Katalin Susztak✉️ (2022) [Epigenomic and transcriptomic analyses define core cell types, genes and targetable mechanisms for kidney disease](https://www.nature.com/articles/s41588-022-01097-w). Nature Genetics, 54(7):950–962. 
  
For more information: [Paper](https://www.nature.com/articles/s41588-022-01097-w); [Research Briefing](https://www.nature.com/articles/s41588-022-01098-9);  [Research Highlight](https://www.kidney-international.org/article/S0085-2538(22)00903-6/fulltext); [GWAS](https://susztaklab.com/GWAS/index.php); [eQTL](http://www.susztaklab.com/Kidney_eQTL/index.php); [meQTL](http://www.susztaklab.com/Kidney_meQTL/index.php); [snATAC-seq](http://www.susztaklab.com/Human_snATAC/index.php); [Github](https://github.com/hbliu/Kidney_Epi_Pri)




