---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

You can also find an up-to-data list of publications on my [Google Scholar profile](https://scholar.google.com/citations?hl=en&user=sM-dRkIAAAAJ&view_op=list_works&sortby=pubdate)

Selected Publications
======
(#Contributed equally; &Corresponding author)
* **Liu H**, Doke T, Guo D, Sheng X, Ma Z, Park J, Vy H, Nadkarni G, Abedini A, Miao Z, Palmer M, Voight B, Li H, Brown C, Ritchie M, Shu Y and Susztak K&. (2022) [Epigenomic and transcriptomic analyses define core cell types, genes and targetable mechanisms for kidney disease](https://www.nature.com/articles/s41588-022-01097-w). Nature Genetics, 54(7):950–962.
![NG2022](/images/pubfig/NG2022.png "NG2022")
* Guan Y#, **Liu H**#, Ma Z, Li SY, Park J, Sheng X and Susztak K&. (2020) Dnmt3a and Dnmt3b-Decommissioned Fetal Enhancers are Linked to Kidney Disease. Journal of the American Society of Nephrology, 31(4), 765-782. 
<img src="/images/pubfig/JASN2020.jpg" width="200">
* Xu S#, **Liu H**#, Wan L#, Zhang W, Wang Q, Zhang S, Shang S, Zhang Y& and Pang D&. (2019) The MS-lincRNA landscape reveals a novel lincRNA BCLIN25 that contributes to tumorigenesis by upregulating ERBB2 expression via epigenetic modification and RNA–RNA interactions in breast cancer. Cell Death & Disease, 10(12), 920.
* Xiong Y#, Wei Y#, Gu Y#, Zhang S, Lyu J, Zhang B, Chen C, Zhu J, Wang Y, **Liu H**&, Zhang Y& (2017) DiseaseMeth version 2.0: a major expansion and update of the human disease methylation database. Nucleic Acids Research, 45(D1):D888-895. 
* **Liu H**&#, Liu X#, Zhang S#, Lv J, Li S, Shang S, Jia S, Wei Y, Wang F, Su J, Wu Q, and Zhang Y& (2016) Systematic identification and annotation of human methylation marks based on bisulfite sequencing methylomes reveals distinct roles of cell-type-specific hypomethylation in regulation of cell identify genes. Nucleic Acids Research, 44(1):75-94.
* Wei Y#, Zhang S#, Shang S#, Zhang B, Wang X, Li S, **Liu H**&, Zhang Y& (2016) SEA: a Super-Enhancer Archive. Nucleic Acids Research, 44(D1):D172-179. 
* **Liu H**#, Zhu R#, Lv J#, He H, Yang L, Huang Z, Su J, Zhang Y, Yu S and Wu Q&. (2014) DevMouse, the mouse developmental methylome database and analysis tools. Database, 2014:bat084. 
* Lv J#, Huang Z#, Liu H#, **Liu H**#, Cui W, Li B, He H, Guo J, Liu Q and Zhang Y and Wu Q&. (2014) Identification and characterization of long intergenic non-coding RNAs related to mouse liver development. Molecular Genetics and Genomics, 289(6), 1225-1235. 
* **Liu H**#, Chen Y#, Lv J#, Zhu R, Su J, Liu X, Zhang Y&, Wu Q& (2013) Quantitative epigenetic co-variation in CpG islands and co-regulation of developmental genes. Scientific Reports, 3:2576. ¬¬
* Lv J#, **Liu H**#, Huang Z#, Su J, He H, Xiu Y, Zhang Y and Wu Q& (2013) Long non-coding RNA identification over mouse brain development by integrative modeling of chromatin and genomic features. Nucleic Acids Research, 41, 10044-10061. 
* Lv J#, Cui W#, **Liu H**#, He H, Xiu Y, Guo J, **Liu H**, Liu Q, Zeng T, Chen Y and Wu Q&. (2013) Identification and characterization of long non-coding RNAs related to mouse embryonic brain development from available transcriptomic data. PloS One, 8, e71152.
* Lv J#, **Liu H**#, Su J#, Wu X, Li B, Xiao X, Wang F, Wu Q&, Zhang Y& (2012) DiseaseMeth: a human disease methylation database. Nucleic Acids Research, 40, D1030-1035. 
* Zhang Y&#, **Liu H**#, Lv J#, Xiao X, Zhu J, Liu X, Su J, Li X, Wu Q, Wang F and Cui Y (2011) QDMR: a quantitative method for identification of differentially methylated regions by entropy. Nucleic Acids Research, 39(9):e58.
* Zhang Y&#, Lv J#, **Liu H**#, Zhu J, Su J, Wu Q, Qi Y, Wang F and Li X&: (2010) HHMD: the human histone modification database. Nucleic Acids Research, 38, D149-154.


All Publications by year
======
<iframe
  src="https://bibbase.org/show?bib=https://bibbase.org/f/CXBcDH3rf8Ee4QN8p/Papers.bib"
  frameborder="0"
  style="width:100%; height:800;"
></iframe>

<script src="https://bibbase.org/show?bib=https://bibbase.org/f/CXBcDH3rf8Ee4QN8p/Papers.bib"></script> 

