---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---
<base target="_blank">

Genetic architecture and epigenetic regulation of kidney disease
------
(Postdoctoral research, 2018 - present)
| I discovered that epigenome (DNA methylation) explains a larger fraction of heritability than gene expression by integrating large scale GWAS, eQTL and meQTL datasets. To further identify disease-causal genes, I proposed a multi-stage prioritization strategy and prioritized >500 kidney disease genes, including <em>SLC47A1<em>, whose causal role was defined in knockout mice model and in human individuals carrying loss-of-function variants. (Liu et al., 2022 [Nature Genetics](https://www.nature.com/articles/s41588-022-01097-w); [Project page](https://susztaklab.com/GWAS/index.php); [Github](https://github.com/hbliu/Kidney_Epi_Pri)). | [<img src="/images/pubfig/NG2022.png">](https://www.nature.com/articles/s41588-022-01097-w) |
| [<img src="/images/pubfig/JASN2020.jpg">](https://jasn.asnjournals.org/content/31/4/765) | To explore epigenetic regulation in kidney development, we profiled DNA methylation by Whole-Genome Bisulfite Sequencing in mouse and human fetal kidneys, and demonstrated essential functions of DNA methylation in decommissioned fetal enhancers linking to kidney disease (Guan, Liu et al. 2020 [JASN](https://jasn.asnjournals.org/content/31/4/765); Liu et al. In preparation). These studies highlighted locus-specific convergence of genetic, epigenetic, and developmental elements in kidney disease development. |
| To identify cell types causally associated with kidney disease, we generated single-nucleus transposase-accessible chromatin with sequencing (snATAC-seq), single-cell RNA sequencing (scRNA), and spatially resolved transcriptomics in mouse and human kidneys. The analysis of these single-cell resolution datasets illustrated the crucial roles of kidney cells (e.g., proximal tubule cells) and cell type-specific genes (e.g., SLC47A1) in kidney injury and fibrosis. (Liu et al., 2022 [Nature Genetics](https://www.nature.com/articles/s41588-022-01097-w); Sheng,…,Liu et al., 2021 [Nature Genetics](https://www.nature.com/articles/s41588-021-00909-9); Miao,…, Liu et al. 2021, [Nature Comms](https://www.nature.com/articles/s41467-021-22266-1); Doke,…, Liu et al., 2021 [JCI](https://www.jci.org/articles/view/141801); Dhillion,…, Liu et al., [Cell Metabolism](https://www.sciencedirect.com/science/article/pii/S1550413120306069?via%3Dihub); Amin,…, Liu et al. 2022 [BioRxiv](https://www.biorxiv.org/content/10.1101/2022.10.24.513598v1)). | [<img src="/images/pubfig/snATAC.png">](https://susztaklab.com/Human_snATAC/index.php) |

Tissue-specific and disease-critical epigenetic regulatory elements
------
(Prior research, 2007 - 2017)
* To identify tissue-specific epigenetic regulatory elements, I developed a series of bioinformatic software, such as SMART for de novo identifying tissue/cell-specific methylated regions from whole genome bisulfite sequencing data (Liu et al. 2016 [NAR](https://academic.oup.com/nar/article/44/1/75/2499653); [Project page](http://fame.edbc.org/smart/)); QDMR for identifying differentially methylated regions from array-based methylation data (Zhang, Liu et al. 2011 [NAR](https://academic.oup.com/nar/article/39/9/e58/1254752); [Project page](http://fame.edbc.org/qdmr/)); QDCMR for identifying differential chromatin regions from ChIP-seq data (Liu et al. 2013 [Sci. Rep.](https://www.nature.com/articles/srep02576); [GitHub](https://github.com/hbliu/QDCMR)).
[<img src="/images/pubfig/SMART2016.png" width="600" class="center">](https://academic.oup.com/nar/article/44/1/75/2499653)
* Leveraging these bioinformatic tools and large-scale epigenome data, I identified tissue-specific epigenetic regulatory elements, including DNA methylation, histone modifications, super-enhancers, and long non-coding RNAs, and explored their roles in mammalian development and human diseases (Liu et al. 2016 [NAR](https://academic.oup.com/nar/article/44/1/75/2499653); Liu et al. 2014 [Database](https://academic.oup.com/database/article/doi/10.1093/database/bat084/2633757); Liu et al. 2013 [Sci. Rep.](https://www.nature.com/articles/srep02576); Xu, Liu et al. 2019 [Cell Death & Disease](https://www.nature.com/articles/s41419-019-2137-5); Xiong,…, Liu et al. 2017 [NAR](https://academic.oup.com/nar/article/45/D1/D888/2605746); Wei,…, Liu et al. 2016 [NAR](https://academic.oup.com/nar/article/44/D1/D172/2503054); Lv, Liu et al. 2013 [NAR](https://academic.oup.com/nar/article/41/22/10044/2438380); Lv, Liu et al. 2012 [NAR](https://academic.oup.com/nar/article/40/D1/D1030/2903287); Zhang, Liu et al. 2011 [NAR](https://academic.oup.com/nar/article/39/9/e58/1254752); Zhang, Lv, Liu et al. 2010 [NAR](https://academic.oup.com/nar/article/38/suppl_1/D149/3112313)). 
[<img src="/images/pubfig/SMART2016.png" width="600" class="center">](https://academic.oup.com/nar/article/44/1/75/2499653)
